misk-actions/src/main/kotlin/misk/exceptions/Exceptions.kt

```diff
@@ -23,7 +23,7 @@ open class WebActionException(
    */
   message: String,
   cause: Throwable? = null
-) : Exception(message, cause) {
+) : RuntimeException(message, cause) {
```

Comment:
Comment the reason for this - i.e. java inter-op - otherwise it's likely someone in the future will change it not realising why.